RNWCPP/include/ReactUWP/ReactRootView.h

```diff
@@ -38,7 +38,7 @@ class ReactRootView : public IXamlRootView
   virtual void SetTag(int64_t tag) override;
 
 private:
-  std::unique_ptr<ReactControl> m_pimpl;
+  std::shared_ptr<ReactControl> m_pimpl;
```

Comment:
I think you also need to update m_pInnerControl to be stored & created as shared_ptr  in ReactControl_rt.* for the winrt control (tested via the C# app)

Reply:
good catch